bin/build-project.ps1

```diff
@@ -55,9 +55,9 @@ Write-Verbose -Message "Building the library with command $dotnet build $Project
 
 if ($MsbuildTarget -eq 'Build') {
     if ($PSVersionTable.PSVersion.Major -ge 6) {
-        $dll = (Resolve-Path -Path "$libraryBase\netcoreapp2.1\dbatools.dll" -ErrorAction Ignore)
+        $dll = (Resolve-Path -Path "$libraryBase\netcoreapp3.1\dbatools.dll" -ErrorAction Ignore)
     } else {
-        $dll = (Resolve-Path -Path "$libraryBase\net452\dbatools.dll" -ErrorAction Ignore)
+        $dll = (Resolve-Path -Path "$libraryBase\net462\dbatools.dll" -ErrorAction Ignore)
```

Comment:
Did you rebuild the library?

Reply:
I did, then committed the change. But I used Visual Studio, not any of our scripts.

Reply:
Yes indeed